plugins/org.eclipse.elk.core.service/src/org/eclipse/elk/core/service/util/CompoundGraphElementVisitor.java

```diff
@@ -0,0 +1,106 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0  
+ *******************************************************************************/
+package org.eclipse.elk.core.service.util;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.elk.core.util.ElkUtil;
+import org.eclipse.elk.core.util.IGraphElementVisitor;
+import org.eclipse.elk.graph.ElkGraphElement;
+import org.eclipse.elk.graph.ElkNode;
+
+/**
+ * Class to apply many graph element visitors on the same graph.
```

Comment:
"on" -> "to".

I'd mention that this is necessary to use more than a single visitor in a layout run with the `DiagramLayoutEngine`.

Reply:
Ok.